### DISCUSSION TITLE
New here. 1.4?

### DISCUSSION THREAD

**Discussion by varrix** [Pryaxis/TShock#1746] [General]
New here. I assume there will be a period in which this must be updated for Terraria 1.4, is this expected to be long (I would assume so as I hear this is a large update)? Just curious!

> Disclaimer:
I don't really play Terraria, but I have friends who are going to be playing the new release tomorrow so I've set them up a server. I went with TShock as it seemed to be the best modded server option.

**Comment by hakusaro** [comment]
I think we've covered most of the 1.4 questions in #1704 but depending on how sizeable the release is, we may or may not be able to do it quickly. There may be experimental versions of TShock within hours or days, and a stable release probably in a couple weeks max.

**Reply by varrix** [reply]
Awesome thanks!
